Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -582,7 +582,6 @@ - (void)updateNote:(Note *)note {
     // push off updating note text in order to speed up animated transition
     dispatch_async(dispatch_get_main_queue(), ^{
         _noteEditorTextView.attributedText = [note.content attributedString];
-        _noteEditorTextView.contentOffset = CGPointMake(0, -_noteEditorTextView.contentInset.top);
```

Comment:
We should probably keep this one for iOS < 11 devices. Maybe wrap this one around an `if #available` check?